ui/hooks/useIsOriginalNativeTokenSymbol.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import fetchWithCache from '../../shared/lib/fetch-with-cache';
+import { BUILT_IN_NETWORKS } from '../../shared/constants/network';
+
+export function useIsOriginalNativeTokenSymbol(chainId, ticker, type) {
+  const [isOriginalNativeSymbol, setIsOriginalNativeSymbol] = useState(null);
+
+  useEffect(() => {
+    async function getNativeTokenSymbol(networkId) {
+      try {
+        const matchedSymbolBuiltInNetwork =
+          BUILT_IN_NETWORKS[type]?.ticker ?? null;
+        if (matchedSymbolBuiltInNetwork) {
+          setIsOriginalNativeSymbol(matchedSymbolBuiltInNetwork === ticker);
+          return matchedSymbolBuiltInNetwork === ticker;
+        }
+
+        const safeChainsList = await fetchWithCache({
```

Comment:
I just remembered another change we should make. We should ensure this gets cached at a long interval. Probably we don't need to make a request more than once per day.

You can see `ui/pages/swaps/swaps.util.ts` for examples of how to set the `cacheRefreshTime` when calling `fetchWithCache`. Then you can import the `DAY` constant from `shared/constants/time.ts`

Reply:
done :)